test/metabase/db/util_test.clj

```diff
@@ -0,0 +1,134 @@
+(ns metabase.db.util-test
+  (:require
+   [clojure.set :as set]
+   [clojure.test :refer [deftest testing is]]
+   [metabase.db.util :as mdb.u]
+   [metabase.models.setting :refer [Setting]]
+   [metabase.util :as u]
+   [toucan2.core :as t2])
+  (:import
+   (java.util.concurrent CountDownLatch)))
+
+(set! *warn-on-reflection* true)
+
+(defn- repeat-concurrently [n f]
```

Comment:
I think this is generally useful, maybe it'll help simplify existing concurrent tests, get rid of Thread/sleeps, or encourage us to write me.

Will just leave it tucked away here for now.